README.md

```diff
@@ -1 +1,192 @@
-# fullstack-assignment
\ No newline at end of file
+# Fullstack Assignment <!-- omit in toc -->
+This project allows de Bijenkorf's Hiring Managers to screen potential Fullstack Developer applicants.
+
+# Table of Contents <!-- omit in toc -->
+- [Folder structure](#folder-structure)
+- [Pre-requisites](#pre-requisites)
+- [Goal](#goal)
+- [Instructions](#instructions)
+- [Requirements](#requirements)
+  - [Scenario: Reservation Overview](#scenario-reservation-overview)
+    - [Loading](#loading)
+    - [No reservations](#no-reservations)
+    - [Reservations](#reservations)
+  - [Scenario: Reservation Detail](#scenario-reservation-detail)
+  - [Scenario: Create Reservation](#scenario-create-reservation)
+    - [Valid input](#valid-input)
+    - [Invalid input](#invalid-input)
+    - [Cancel](#cancel)
+  - [Scenario: Edit Reservation](#scenario-edit-reservation)
+    - [Valid input](#valid-input-1)
+    - [Invalid input](#invalid-input-1)
+    - [Cancel](#cancel-1)
+  - [Scenario: Delete Reservation](#scenario-delete-reservation)
+- [Notes](#notes)
+- [Submitting your code](#submitting-your-code)
+
+# Folder structure
+The initial project structure looks like this:
+```
+./client
+./server
+```
+
+You will find further information in the respective documentation for each directory.
+
+# Pre-requisites
+You will can open the wireframes in [Draw.io](https://app.diagrams.net/) in your browser or by using the [Draw.io VSCode extension](https://marketplace.visualstudio.com/items?itemName=hediet.vscode-drawio)
+
+# Goal
+Create a straight forward web application that demonstrates knowledge of the following:
+- Data storage;
+- API CRUD operations;
+- Client state management;
+- Client routing;
+- Implementation of UI in React;
+- Unit Testing
+
+# Instructions
+You are asked to create a working application. Technology choice is mostly up to you, however we ask you to implement the View Layer in [React](https://reactjs.org/). For that reason we have bootstrapped the [client](./client) with [Create React App](https://create-react-app.dev/). Implementation of routing, state management and networking is down to you. While you are free to implement the API in any way you feel appropriate, we have setup a mock JSON API [server](./server) with [json-server](https://github.com/typicode/json-server) as an option.
```

Comment:
As the choice of implementing back-end is arbitrary and only usage of json-server would be sufficient for the scenarios that are written here. It looks like this is front-end focused assignment. 
Could there be some room for the implementation of some back-end scenarios as well?